### PR TITLE
Add multiple Hook registration usage

### PR DESCRIPTION
**Pull request by PululuK**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add multiple Hook registration usage
| Fixed ticket? | See https://github.com/PrestaShop/PrestaShop/issues/21423#issuecomment-707759918

### CODE REVIEW

File: src/content/1.7/development/components/hook/_index.md [hunk 1]
```diff
@@ -31,6 +31,27 @@ public class somemodule extends Module
...
+    {
+        $this->name = 'somemodule';
+        $this->hooks = ['registerGDPRConsent', 'displayProductAdditionalInfo'];
+    }  
+    
+    public function install()
+    {
+        return parent::install() && $this->registerHook($this->hooks);
+    }
+}
```
Comment (PierreRambaud) [inline src/content/1.7/development/components/hook/_index.md:52]:
```suggestion
<?php
class somemodule extends Module
{
    const AVAILABLE_HOOKS = [
        'registerGDPRConsent',
        'displayProductAdditionalInfo',
    ];
    
    public function __construct()
    {
        $this->name = 'somemodule';
    }  
    
    public function install()
    {
        return parent::install() && $this->registerHook(self::AVAILABLE_HOOKS);
    }
}
```

File: src/content/1.7/development/components/hook/_index.md [hunk 2]
```diff
@@ -31,6 +31,31 @@ public class somemodule extends Module
     }
 }
 ```
+You can also register multiple hooks. Here is how a Module subscribes to hook `registerGDPRConsent` and `displayProductAdditionalInfo`:
+
+
+```php
+<?php
```
Comment (PierreRambaud) [inline src/content/1.7/development/components/hook/_index.md:38]:
```suggestion
```

File: src/content/1.7/development/components/hook/_index.md [hunk 3]
```diff
@@ -31,6 +31,30 @@ public class somemodule extends Module
     }
 }
 ```
+You can also register multiple hooks. Here is how a Module subscribes to hook `registerGDPRConsent` and `displayProductAdditionalInfo`:
```
[the remark this answers is not in the document]
Comment (PululuK) [reply src/content/1.7/development/components/hook/_index.md:34]: Yep !

[the remark this answers is not in the document]
Comment (PierreRambaud) [reply src/content/1.7/development/components/hook/_index.md:34]: The one on line 26 :)

[the remark this answers is not in the document]
Comment (PululuK) [reply src/content/1.7/development/components/hook/_index.md:34]: humm .. Why  @PierreRambaud  ?

**Comment by PululuK** [comment]
> @PululuK maybe `CamelCase`?

Yep ... thanks 🖤

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
> > @PululuK maybe `CamelCase`?
> 
> Yep ... thanks

So much love :heart_eyes_cat:

**Review by matks — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @PululuK

**Comment by onklmaps** [comment]
Thank you ❤️

**Pull request merged**